libcxx/include/stack

```diff
@@ -317,14 +317,14 @@ public:
 
 #if _LIBCPP_STD_VER >= 17
 template<class _Container,
-         class = enable_if_t<!__is_allocator<_Container>::value>
+         class = enable_if_t<!__is_allocator<_Container> >
```

Comment:
Nit: This is C++17 code, so we don't need the whitespace. Same for quite a few other places.